data/json/items/tool_armor.json

```diff
@@ -2129,5 +2129,21 @@
         "encumbrance" : 5,
         "coverage" : 10,
         "material_thickness" : 2
-    }
+    },
+  {
```

Comment:
JSON style, aim for consistency with either existing file or new linter format. The below is neither and also inconsistent with itself.